Gruntfile.js

```diff
@@ -166,6 +166,14 @@ module.exports = function (grunt) {
       }
     },
 
+    // ESLint is a replacement for JSHint + includes style linting + is extensible
+    eslint: {
+        target: [
+          'Gruntfile.js',
+          '<%= yeoman.app %>/scripts/{,*/}*.js'
+        ]
+    },
```

Comment:
I worry about having to fix a bunch of style warnings when I need to change one line to fix a bug. Particularly when back-porting fixes where you want to minimize changes. I'd prefer to either:

1. Bite the bullet and update all scripts upfront, then make eslint required.
2. Make eslint failures warnings and fix things where it makes sense as we're in the code.

Probably (2) if there are a lot of existing problems.